### PR TITLE
Add S3 auth example to Kafka source

### LINKED ISSUE #628: Bug: lacking a example to load schema from s3
### Where's the bug?

https://www.risingwave.dev/docs/current/create-source-kafka

### Describe the bug

> RisingWave supports reading schemas from a Web location in `http://...`, `https://...`, or `S3://...` format, or a Confluent Schema Registry for Kafka data in Avro or Protobuf format.

when loading files from s3, we may face loading a file from a bucket requiring authorization. It is helpful if we provide an example for demonstration.

### PR DESCRIPTION
**Pull request by emile-00**
## Info
- **Description**: 
Add S3 auth connection parameters and example to Ingest data from Kafka page.

- **Preview**: 
[ Paste the preview link to the edited page here. Edit this item after you submit the PR and the preview is ready. ]

- **Related code PR**: 

- **Related doc issue**: 
Resolves https://github.com/risingwavelabs/risingwave-docs/issues/628

- **Notes**: 
[ Any additional information? ]

## Before merging
  - [ ] (For version-specific PR) I have selected the corresponding software version in **Milestone** and linked the related doc issue to this PR in **Development**.
  - [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`bernscode`, `CharlieSYH`, `emile-00`, & `hengm3467`). 
  - [ ] I have checked the doc site preview, and the updated parts look good. <details><summary>How?</summary>Scroll down and open this link: image
</details>

### CODE REVIEW

File: docs/create-source/create-source-kafka.md [hunk 1]
```diff
@@ -262,6 +262,35 @@ To specify a schema location, add this clause to a `CREATE SOURCE` statement.
 ROW SCHEMA LOCATION 'location'
 ```
 
+If you are loading files from an S3 bucket, additional authorization may be required. Use the following additional parameters in your `CREATE SOURCE` or `CREATE TABLE` statement for S3 authorization.
```
Comment (hengm3467) [inline docs/create-source/create-source-kafka.md]: @emile-00 I think it's about schema files in S3 buckets, not it's better to spell it out.

File: docs/create-source/create-source-kafka.md [hunk 2]
```diff
@@ -262,6 +262,35 @@ To specify a schema location, add this clause to a `CREATE SOURCE` statement.
...
+   connector='kafka',
+   topic='demo_topic',
+   properties.bootstrap.server='172.10.1.1:9090,172.10.1.2:9090',
+   scan.startup.mode='latest',
+   scan.startup.timestamp_millis='140000000'
+   access_key='replace me with your aws access key',
+   secret_access='replace me with your aws secret access'
+)
+ROW FORMAT AVRO 
+ROW SCHEMA LOCATION CONFLUENT SCHEMA REGISTRY 'http://127.0.0.1:8081';
```
Comment (hengm3467) [inline docs/create-source/create-source-kafka.md:291]: @TabVersion If Confluent Schema Registry is used, is it still necessary to get schemas from S3. I think the line at the bottom of the example here should be 
`ROW FORMAT LOCATION 'S3://...'`
Correct?

Comment (tabVersion) [reply docs/create-source/create-source-kafka.md:291]: we only load schema from one place. If schema registry is provided, we don't fetch s3 file.
Also we are considering removing loading schema file from s3, cc @neverchanje

Comment (hengm3467) [reply docs/create-source/create-source-kafka.md:291]: @tabVersion If this is the case, we can just close this PR and the related doc issue #628. 

@emile-00 for visibility.

**Pull request closed without merging**